Services/UserService/Controllers/UsersController.cs

```diff
@@ -35,9 +35,9 @@ public async Task<bool> DeleteUser([FromServices] IDeleteUserCommand command, [F
 
         [Route("get")]
         [HttpGet]
-        public User GetUser([FromHeader] Guid userId)
+        public async Task<User> GetUser([FromServices] IGetUserByIdCommand command, [FromQuery] Guid userId)

```

Comment:
Why do you use  Guid here and not getUserByIdRequest in RestRequests? 

And why from query and not header?

Reply:
> Why do you use Guid here and not getUserByIdRequest in RestRequests?
> 
> And why from query and not header?

Fixed, my bad